src/test/java/com/squareup/protoparser/ProtoParserTest.java

```diff
@@ -919,6 +926,148 @@
+                .addOption(OptionElement.create("default", Kind.NUMBER, "-2147483648"))
+                .build())
+            .addField(FieldElement.builder()
+                .label(OPTIONAL)
+                .type(ScalarType.INT64)
+                .name("default_int64")
+                .tag(406)
+                .addOption(OptionElement.create("default", Kind.NUMBER, "9223372036854775807"))
+                .build())
+            .addField(FieldElement.builder()
+                .label(OPTIONAL)
+                .type(ScalarType.UINT64)
+                .name("default_uint64")
+                .tag(407)
+                .addOption(OptionElement.create("default", Kind.NUMBER, "18446744073709551615"))
+                .build())
+            .addField(FieldElement.builder()
+                .label(OPTIONAL)
+                .type(ScalarType.SINT64)
+                .name("default_sint64")
+                .tag(408)
+                .addOption(OptionElement.create("default", Kind.NUMBER, "-9223372036854775808"))
+                .build())
+            .addField(FieldElement.builder()
+                .label(OPTIONAL)
+                .type(ScalarType.FIXED64)
+                .name("default_fixed64")
+                .tag(409)
+                .addOption(OptionElement.create("default", Kind.NUMBER, "18446744073709551615"))
+                .build())
+            .addField(FieldElement.builder()
+                .label(OPTIONAL)
+                .type(ScalarType.SFIXED64)
+                .name("default_sfixed64")
+                .tag(410)
+                .addOption(OptionElement.create("default", Kind.NUMBER, "-9223372036854775808"))
+                .build())
+            .addField(FieldElement.builder()
+                .label(OPTIONAL)
+                .type(ScalarType.BOOL)
+                .name("default_bool")
+                .tag(411)
+                .addOption(OptionElement.create("default", Kind.BOOLEAN, "true"))
+                .build())
+            .addField(FieldElement.builder()
+                .label(OPTIONAL)
+                .type(ScalarType.FLOAT)
+                .name("default_float")
+                .tag(412)
+                .addOption(OptionElement.create("default", Kind.NUMBER, "123.456e7"))
+                .build())
+            .addField(FieldElement.builder()
+                .label(OPTIONAL)
+                .type(ScalarType.DOUBLE)
+                .name("default_double")
+                .tag(413)
+                .addOption(OptionElement.create("default", Kind.NUMBER, "123.456e78"))
+                .build())
+            //.addField(FieldElement.builder()
+            //    .label(OPTIONAL)
+            //    .type(ScalarType.STRING)
+            //    .name("default_string")
+            //    .tag(414)
+            //    .addOption(OptionElement.create("default", Kind.STRING,
+            //        "çok\\a\\b\\f\\n\\r\\t\\v\\1\\01\\001\\17\\017\\176\\x1\\x01\\x11\\X1\\X01\\X11güzel"))
```

Comment:
I think you want the equals() for `Kind.STRING` to be equal after conversion. Which means you probably need to decode it when you create the option.